src/gui/properties/downloadedpiecesbar.cpp

```diff
@@ -28,216 +28,144 @@
  * Contact : chris@qbittorrent.org
  */
 
+#include "downloadedpiecesbar.h"
 #include <cmath>
 #include <QDebug>
-#include "downloadedpiecesbar.h"
 
-DownloadedPiecesBar::DownloadedPiecesBar(QWidget *parent): QWidget(parent)
+DownloadedPiecesBar::DownloadedPiecesBar(QWidget *parent)
+    : base(true, parent)
+    , m_dlPieceColor{0, 0xd0, 0}
```

Comment:
Fix coding style!
We require extra parenthesis in constructor initialization list.
And also fix spaces in `{}`:
```c++
    , m_dlPieceColor({ 0, 0xd0, 0 })
```